ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -849,7 +849,7 @@ import '../nuxeo-button-styles.js';
 
     _onCheckBoxTap(e) {
       if (this.selectionEnabled && !this.selectAllActive) {
-        this.$.list.toggleSelectionForIndex(e.model.index);
+        // this.$.list.toggleSelectionForIndex(e.model.index);
```

Comment:
Should we add more information about why we're commenting this line out?

Reply:
I think we should remove the line and maybe add a comment explaining the behavior. wdyt?